clogMonitor/src/screens/DashboardScreen.js

```diff
@@ -62,14 +62,14 @@ function DashboardScreen({ setLoggedIn }) {
         {/* </div> */}
         <div className="DashboardScreen__Center">
           <div className="DashboardScreen__Center__main">
-            <BrowserRouter>
+            <HashRouter>
               <Routes>
                 <Route path="/" element={<Home logEvents={logEvents} loading={loading} />} />
                 <Route path="business-processes" element={<BusinessView/>} />
```

Comment:
Would be nice for this to be "/business-processes", but not necessary apparently